src/Network/Ethereum/Web3/Contract/Events.purs

```diff
@@ -57,6 +57,7 @@ import Prim.RowList as RowList
 type EventHandler f e
   = e -> ReaderT Change f EventAction
 
+type FilterStreamState :: forall (k :: Type) (e :: k). Type -> Type
```

Comment:
is `(k :: Type` actually necessary? I'm assuming that `(e :: Type)` would be sufficent, but even then I'm surprised they are forcing you to give a kind annotation for these RankNTypes